.azure-pipelines/LinuxAnalyzerTest.yml

```diff
@@ -43,10 +43,10 @@ jobs:
       mkdir -p $HOME/go/bin
       curl https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh
 
-      # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released, make a
-      # build from the latest revision of the master branch.
-      git clone https://github.com/CocoaPods/CocoaPods.git
-      cd CocoaPods
+      # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released.
+      curl -ksSL -o cocoapods.zip https://github.com/CocoaPods/CocoaPods/archive/9461b346aeb8cba6df71fd4e71661688138ec21b.zip
```

Comment:
You could use `-O` (uppercase) instead of `-o cocoapods.zip`. Or yet simpler, pipe `curl`'s output directly to `unzip`. That way you also don't have to manually remove the ZIP file, see the similar code below.

Reply:
Why should `-O` instead of `-o` be used?

The code below pipes to `tar`, not to `unzip` and I was not able to figure out how it works with `unzip`.
Can you give a hint?

Reply:
These lines still use the previous approach.